src/js/main.js

```diff
@@ -496,5 +531,16 @@ jQuery(document).ready(function () {
     } else {
       jQuery('.disable-if-sticky input').removeAttr('disabled');
     }
-  })
+  });
+
+  jQuery('#terms').on('change', function () {
+    var button = jQuery('#btn-register');
+    if (jQuery(this).prop('checked')) {
+      jQuery(button).removeAttr('disabled');
+    } else {
+      jQuery(button).attr('disabled', 'true');
+    }
```

Comment:
- this can be achieved by using `prop()` instead of `attr` (see  -> http://stackoverflow.com/questions/13626517/remove-disabled-attribute-using-jquery):

`jQuery('#btn-register').prop('disabled', !jQuery(this).prop('checked'))`